src/schema/objects/columns.yaml

```diff
@@ -53,9 +53,12 @@ channel:
   display_name: Channel
   description: |
     Channel(s) associated with an event.
-    If multiple channels are specified, separate with commas (`,`) or a
-    delimiter specified in the `"Delimiter"` field describing the `channel`
-    column.
+    If multiple channels are specified, they MUST be separated by a delimiter
```

Comment:
As in the task.yaml, I think that this is confusing.  Is the Delimiter required as line 59 seems to indicate it is, but the earlier changes indicate that it isn't and still include the notion of a default delimiter.